2022-05-12 22:56:14,230 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1571] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 22:59:22,687 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1578] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:01:31,507 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1581] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:01:31,507 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1581] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:16:48,602 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1585] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:16:58,889 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1585] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:16:58,889 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1585] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:34:33,501 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1596] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-12 23:34:33,501 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1596] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:49:11,838 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1624] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:49:11,838 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1624] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:51:35,520 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:52:32,510 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:52:47,422 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:52:47,422 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:56:02,852 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:56:18,546 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 21:56:18,546 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:25:58,015 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: Packet sequence number wrong - got 1 expected 2
2022-05-13 22:26:28,324 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:28:48,379 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:29:17,002 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:29:27,241 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:29:27,241 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:30:01,568 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:40:05,026 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:40:05,026 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:43:29,212 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:43:29,212 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1627] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:45:23,420 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-13 22:45:52,910 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1629] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:15:39,403 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:15:39,403 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:16:09,993 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:16:20,247 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:16:20,247 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:16:39,594 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:16:39,594 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:17:00,797 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:17:00,797 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:17:33,672 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:18:39,203 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 00:20:01,086 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10051] 向一个无法连接的网络尝试了一个套接字操作。)")
2022-05-14 00:20:01,343 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10051] 向一个无法连接的网络尝试了一个套接字操作。)")
2022-05-14 00:20:01,343 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1645] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10051] 向一个无法连接的网络尝试了一个套接字操作。)")
2022-05-14 00:32:45,429 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 00:32:45,429 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 00:32:45,429 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 00:34:21,168 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 00:34:21,168 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 00:34:21,168 - G:\毕设项目\my-graduate-project\API\appTest\pc_api\views.py[line:665] - ERROR: local variable 'res' referenced before assignment
2022-05-14 10:50:39,293 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1704] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10065] 套接字操作尝试一个无法连接的主机。)")
2022-05-14 10:50:39,293 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1704] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10065] 套接字操作尝试一个无法连接的主机。)")
2022-05-14 10:50:39,293 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1704] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' ([WinError 10065] 套接字操作尝试一个无法连接的主机。)")
2022-05-14 12:04:25,457 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1691] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 12:04:36,453 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1691] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 12:04:36,453 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1691] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 23:27:19,268 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1704] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 23:41:48,946 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1764] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
2022-05-14 23:41:48,946 - G:\毕设项目\my-graduate-project\API\appTest\utils\dbV3\db.py[line:1764] - ERROR: (2003, "Can't connect to MySQL server on '43.138.188.22' (timed out)")
